pages/agent/v3/cli_annotate.md.erb

```diff
@@ -57,6 +63,23 @@ Options:
    --debug-http                Enable HTTP debug mode, which dumps all request and response bodies to the log [$BUILDKITE_AGENT_DEBUG_HTTP]
 ```
 
+## Grouping annotations
```

Comment:
Perhaps we leave this section out for now and merge the mention of the 5 annotation build page limit, or look at a "Updating and appending annotations" section instead?

Reply:
Sounds good!